1.6/enterprise/large/fixtures/jobs.yml

```diff
@@ -317,3 +359,87 @@ jobs:
             enclosure: '"'
             withHeader: true
             filePath: /tmp/attribute-option.csv
+    xlsx_product_quick_export:
+        connector: 'Akeneo Mass Edit Connector'
```

Comment:
xlsx_quick_export uses Akeneo Mass Edit Connector ? (don't know, it's a question)